reana_commons/config.py

```diff
@@ -49,7 +49,7 @@
             port=os.getenv('WORKFLOW_CONTROLLER_SERVICE_PORT_HTTP', '5000')),
         'reana_workflow_controller.json'),
     'reana-server': (
-        os.getenv('REANA_SERVER_URL', None),
+        os.getenv('REANA_SERVER_URL', '0.0.0.0'),
```

Comment:
LGTM, please fix commit log message formatting (missing headline!) and add yourself to the author list.